client/galaxy/scripts/layout/masthead.js

```diff
@@ -81,34 +71,38 @@ const View = Backbone.View.extend({
     },
 
     render: function() {
-        let brand = this.options.display_galaxy_brand ? "Galaxy " : "";
+        const el = document.createElement("div");
+        this.el.appendChild(el); // use this.el directly when feature parity is accomplished
+        let brandTitle = this.options.display_galaxy_brand ? "Galaxy " : "";
         if (this.options.brand) {
-            brand += this.options.brand;
+            brandTitle += this.options.brand;
         }
-        this.$navbarBrandTitle.html(brand);
-        this.$navbarBrandLink.attr("href", this.options.logo_url);
-        this.$navbarBrandImage.attr("src", this.options.logo_src);
-        this.quotaMeter.render();
+        mountVueComponent(Masthead)(
+            {
+                brandTitle: brandTitle,
+                brandLink: this.options.logo_url,
+                brandImage: this.options.logo_src,
+                quotaMeter: this.quotaMeter,
+                activeTab: () => {
```

Comment:
Here we could parse `this.activeView` directly instead of wrappig it into a function.